hiring/rationale.md

```diff
@@ -0,0 +1,15 @@
+# Rationale
+
+We believe that our hiring process should be fair and repeatable. What do we mean by this?
+
+## Fair
+
+We want to ensure our process does not bias any one particular type of person be that industry experience, race, gender, religion, disability, introvert, extrovert or any other categorisation. There is a simple reason to this, we know that having people of all different walks of life provide us with new view points and better our ability to achieve our mission.
```

Comment:
I'm over people having to try and list every kind of 'categorisation'. It should be fine to say something like:

> Made is an equal opportunity employer that is committed to inclusion and diversity. There is a simple reason to this, we know that having people of all different walks of life provide us with new view points and better our ability to achieve our mission.

A lot simpler, and puts an emphasis on us wanting diversity.